ratatool-sampling/src/main/scala/com/spotify/ratatool/samplers/BigSamplerBigQuery.scala

```diff
@@ -18,14 +18,18 @@
 package com.spotify.ratatool.samplers
 
 import java.util.{List => JList}
-
 import com.google.api.services.bigquery.model.{TableFieldSchema, TableReference}
 import com.google.common.hash.Hasher
-import com.spotify.ratatool.samplers.util.{ByteEncoding, Precision, RawEncoding, SampleDistribution}
+import com.spotify.ratatool.samplers.util.{ByteEncoding,
+  HashAlgorithm,
+  Precision,
+  RawEncoding,
+  SampleDistribution}
 import com.spotify.scio.ScioContext
 import com.spotify.scio.bigquery.TableRow
 import com.spotify.scio.io.ClosedTap
-import org.apache.beam.sdk.io.gcp.bigquery.{BigQueryIO, BigQueryOptions,
+import org.apache.beam.sdk.io.gcp.bigquery.{BigQueryIO,
```

Comment:
Nitpick since this is a bit of style preference, but there's no real reason to change this line at the moment except to match the style of the above line which we should change anyway.

Reply:
Reverted